src/panels/lovelace/cards/hui-entity-filter-card.ts

```diff
@@ -36,6 +36,10 @@ class EntityFilterCard extends HTMLElement implements LovelaceCard {
       throw new Error("Incorrect filter config.");
     }
 
+    if (!config.entities) {
```

Comment:
Place this check above the check for filter config, and check if it is an array, as we assume in that check entities is an array.